### PR TITLE
Added the PictureFill polyfill for the Picture element

### PR DESCRIPTION
**Pull request by jonathan-fielding**
Hi

This is a pull request to add support for the Picture element in unsupported browsers using the Polyfill service

Thanks

Jonathan

### CODE REVIEW

File: docs/assets/compat.json [hunk 1]
```diff
@@ -3007,6 +3007,41 @@
...
+      "7": "polyfilled"
+    },
+    "android": {
+      "4.4": "polyfilled",
+      "4.3": "polyfilled",
+      "4.2": "polyfilled",
+      "4.1": "polyfilled",
+      "4.0": "polyfilled"
+    }
+  },
```
Comment (triblondon) [inline docs/assets/compat.json:3313]: Don't edit this file.  It gets generated automatically by automation.

File: polyfills/Picture/config.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		"ios_saf": "*"
+	},
+	"dependencies": [
+	],
+	"license": "MIT",
+	"spec": "https://html.spec.whatwg.org/multipage/embedded-content.html#embedded-content",
+	"repo": "https://github.com/scottjehl/picturefill",
+	"notes": [
+		"Standard picturefill polyfill used to enable support in older browsers"
+	],
```
Comment (triblondon) [inline polyfills/Picture/config.json]: This note could be removed - it doesn't signal any important gotchas

**Comment by triblondon** [comment]
Thanks Jonathan.  Aside from the minor issues above, are there any tests that can be added?  We like to have a test for at least the main core use cases, so that our test framework can check browser compatibility.

**Comment by triblondon** [comment]
Also, it would be good to have an update task.  This is a new feature that enables third party polyfills to be kept up to date automatically.  I've just pushed some documentation for how to create an update task in https://github.com/Financial-Times/polyfill-service/commit/d7cc2cd0a2a1d7afbad6b706d77172502080433d.

There's a good example of an existing task in https://github.com/Financial-Times/polyfill-service/blob/master/polyfills/Intl/update.task.js

File: docs/assets/compat.json (hunk 1 above)
Comment (jonathan-fielding) [inline docs/assets/compat.json:3313]: If this is generated automatically is there any reason why it is not in the .gitignore?

Comment (triblondon) [inline docs/assets/compat.json:3313]: it needs to be committed - it's used to display the compatibility chart in
the documentation.  I suppose we could add it during deployment but the
full compat test takes about 2 hours, so I'd rather not block deploys on
that.

Comment (jonathan-fielding) [inline docs/assets/compat.json:3313]: ok removed my changes

File: polyfills/Picture/config.json (hunk 2 above)
Comment (jonathan-fielding) [inline polyfills/Picture/config.json]: removed note

**Comment by jonathan-fielding** [comment]
Update task has been added

File: polyfills/Picture/update.task.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	// deleting existing files
+	if (grunt.file.exists(PictureFillPolyfillOutput + 'polyfill.js')) {
+		try {
+			// Actually delete. Or not.
+			rimraf.sync(PictureFillPolyfillOutput + 'polyfill.js');
+		} catch (e) {/* ignore */}
+    }
+
+	// customizing the config to add Picture as a dependency
+	configSource.dependencies.push('Picture');
```
Comment (JakeChampion) [inline polyfills/Picture/update.task.js]: I think the dependency might be the ~html5-elements/polyfill.js

Comment (jonathan-fielding) [inline polyfills/Picture/update.task.js]: Just committed the change, is that what you mean? https://github.com/Financial-Times/polyfill-service/commit/34bfe544cf59d825250bccfbc3b4d83bb1ae62b2

**Comment by triblondon** [comment]
Implemented in #448

**Pull request closed without merging**